descwl_coadd/coadd.py

```diff
@@ -79,7 +79,7 @@ def make_coadd_obs(
     CoaddObs (inherits from ngmix.Observation)
     """
 
-    coadd_exp, coadd_noise_exp, coadd_psf_exp = make_coadd(
+    coadd_exp, coadd_noise_exp, coadd_psf_exp, coadd_mfrac_exp = make_coadd(
```

Comment:
this return list is getting long.  should we convert to dict to avoid bugs?

Reply:
That's a good idea. People also like named tuples for this kind of thing, but I find dicts more intuitive.

Reply:
done.